src/test/java/com/sap/oss/phosphor/fosstars/data/release/info/ReleaseInfoLoaderTest.java

```diff
@@ -0,0 +1,258 @@
+package com.sap.oss.phosphor.fosstars.data.release.info;
+
+import static com.sap.oss.phosphor.fosstars.model.feature.oss.OssFeatures.RELEASED_ARTIFACT_VERSIONS;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.when;
+
+import com.sap.oss.phosphor.fosstars.data.github.ReleasesFromGitHub;
+import com.sap.oss.phosphor.fosstars.data.github.TestGitHubDataFetcherHolder;
+import com.sap.oss.phosphor.fosstars.model.subject.oss.GitHubProject;
+import com.sap.oss.phosphor.fosstars.model.subject.oss.MavenArtifact;
+import com.sap.oss.phosphor.fosstars.model.subject.oss.NpmArtifact;
+import com.sap.oss.phosphor.fosstars.model.value.ValueHashSet;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+import org.apache.commons.io.IOUtils;
+import org.apache.http.HttpEntity;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.junit.Before;
+import org.junit.Test;
+import org.kohsuke.github.GHRelease;
+import org.kohsuke.github.GHRepository;
+import org.kohsuke.github.PagedIterable;
+
+public class ReleaseInfoLoaderTest extends TestGitHubDataFetcherHolder {
```

Comment:
The test cases look like duplicates. How about focusing on the logic of `ReleaseInfoLoader`? The test could check if right providers are called when:

- a Maven artifact is passed
- an NPM artifact is passed
- unknown artifact is passed
- fallback to `ReleasesFromGitHub` happens when necessary

Maybe you can just mock `update()` methods of the underlying providers to check the logic in `ReleaseInfoLoader`.